### PR TITLE
Fix small bugs

### PR DESCRIPTION
**Pull request by wouterbaltus**
- Fixed some general styling
- Fixed select view delay
- Removed skeleton row hover

### CODE REVIEW

File: src/components/data-grid/FooterPageSize.tsx [hunk 1]
```diff
@@ -2,9 +2,9 @@ import React, { useEffect, useState } from 'react';
 import { FooterRowCountProps } from './types';
 import { StyledFooterFooterPageSize } from './styles';
 import { Select } from '../select/Select';
+import ParagraphSmall from '../typography/ParagraphSmall';
```
Comment (hiddeottens) [inline src/components/data-grid/FooterPageSize.tsx:5]: Prefer named imports like it used to be

`import { ParagraphSmall } from '../typography';`

File: src/components/data-grid/views/DataGridViews.tsx [hunk 2]
```diff
@@ -69,16 +70,30 @@ export const DataGridViews = ({
...
+
     if (onActivateView) {
       await onActivateView(id);
     }
   };
 
+  const isActiveView = (id: string) => {
+    if (activeViewId) {
+      return activeViewId === id;
+    } else {
```
Comment (hiddeottens) [inline src/components/data-grid/views/DataGridViews.tsx:83]: unnecessary else statement

**Review by hiddeottens — APPROVED** [review]

File: src/components/data-grid/views/DataGridViews.tsx (hunk 2 above)
Comment (wouterbaltus) [reply src/components/data-grid/views/DataGridViews.tsx]: Fixed

File: src/components/data-grid/FooterPageSize.tsx (hunk 1 above)
Comment (wouterbaltus) [reply src/components/data-grid/FooterPageSize.tsx]: Fixed

**Pull request merged**